packages/default-wallets/src/lib/default-wallets.ts

```diff
@@ -0,0 +1,22 @@
+import { setupMyNearWallet } from "@near-wallet-selector/my-near-wallet";
+import { setupLedger } from "@near-wallet-selector/ledger";
+import { setupWalletConnect } from "@near-wallet-selector/wallet-connect";
+import type { WalletModuleFactory } from "@near-wallet-selector/core";
+
+export const setupDefaultWallets = async (): Promise<
+  Array<WalletModuleFactory>
+> => {
+  return [
+    setupMyNearWallet(),
+    setupLedger(),
+    setupWalletConnect({
+      projectId: "c8cb6204543639c31aef44ea4837a554",
```

Comment:
hmm -- where did this projectId come from?

Reply:
This is coming from here: https://cloud.walletconnect.com/sign-in .
I have mentioned it a few times on Slack that if we add WalletConnect to the default wallets it will require some params.